x-pack/plugins/code/public/monaco/monaco_helper.ts

```diff
@@ -68,6 +68,7 @@ export class MonacoHelper {
           scrollBeyondLastLine: false,
           renderIndentGuides: false,
           automaticLayout: false,
+          lineDecorationsWidth: 16,
```

Comment:
This hard-wired constant would make it hard to find in the future. can we make it a meaningful constant at the beginning of this file?